packages/slate/test/schema/custom/node-mark-invalid-default.js

```diff
@@ -5,7 +5,7 @@ import h from '../../helpers/h'
 export const schema = {
   blocks: {
     paragraph: {
-      marks: ['bold'],
+      marks: [{ type: 'bold' }, { type: 'underline' }],
```

Comment:
could we avoid breaking the interface here by continuing to pass a string?

Reply:
It seemed to me that the tests were using the wrong interface, as the existing code was comparing to `mark.type`

Reply:
Oh yeah you're right! haha